products/osconfig/api.yaml

```diff
@@ -874,7 +874,7 @@ objects:
                   - rollout.0.disruption_budget.0.fixed
                   - rollout.0.disruption_budget.0.percentage
               - !ruby/object:Api::Type::Integer
-                name: percentage
+                name: percent
```

Comment:
Instead of changing `name:` to `percent`, we could add an `api_name` value of `percent`. The name in Terraform will still be wrong, but it will map to the correct API field.

It's hard to remove an existing nested field in practice- we would need to deprecate the field and then remove it in a major release. It's probably worth keeping the minor deviation from the API.

Reply:
I've updated this line accordingly in the most recent commit